### ISSUE TITLE
Add some convenience shortcuts for specifying basemaps when using ipyleaflet

### ISSUE THREAD

**Issue by alimanfoo** [malariagen/malariagen-data-python#340]
I do this a lot...

```
ag3.plot_samples_interactive_map(basemap=ipyleaflet.basemaps.OpenStreetMap.Mapnik)
```

...but I can never remember the exact location of the Mapnik basemap inside ipyleaflet. It would be easier if I could do this...

```
ag3.plot_samples_interactive_map(basemap="mapnik")
```

...and have similar shortcuts for other basemaps.

**Comment by leehart** [comment]
I suppose we could switch on `type(basemap) == str`.

**Comment by leehart** [comment]
Closed by merged PR #372. We can open separate issues for a dropdown and custom marker icon support.
